src/image/modify-spec.py

```
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)